common/api/summary/presentation-common.exports.csv

```diff
@@ -86,6 +86,9 @@ public;ECPropertyValueQuerySpecification
 public;EditorDescription
 public;EnumerationChoice
 public;EnumerationInfo
+alpha;ExpandedNodeUpdateRecord
+alpha;ExpandedNodeUpdateRecord
```

Comment:
any idea why these are duplicated?

Reply:
I think it's because there are `interface` and `namespace` defined with same name. I can see duplicate lines for `HierarchyCompareInfo` also.

Reply:
thanks